cloud/aws/.env.example

```diff
@@ -0,0 +1,3 @@
+VAST_PEERED_VPC_ID=
```

Comment:
(Not relevant to the line, just wanted to leave a resolvable comment)

Can I ask you to rebase and reword your commit message to adhere to the guidelines laid out in our contributing guide? https://vast.io/docs/develop-vast/contributing/workflow#commit-messages

Reply:
Done